postgresql-containers/build/pgbackrest/Dockerfile

```diff
@@ -102,9 +102,13 @@ RUN set -ex; \
         percona-pgbackrest; \
     microdnf -y clean all
 
-RUN set -ex; \
-    groupadd postgres -g 26; \
-    useradd postgres -u 26 -g 26
+# only PG16 is affected PG-794
+RUN if [[ "${PG_MAJOR//.}" -le '15' ]]; then \
+        groupadd postgres -g 26; \
+        useradd postgres -u 26 -g 26; \
+    else \
```

Comment:
Could we just check exists user or not?

Reply:
we can but in next PG releases we will have this user for all versions and we will just remove this check